ballerina/types.bal

```diff
@@ -68,6 +68,15 @@ public type Varray record {|
     ArrayValueType? elements;
 |};
 
+# An ordered set of data elements with a variable size.
+#
+# + name - Name of the varray
+# + elements - Elements of the Varray
+public type NestedTableType record {|
+    string name;
+    ArrayValueType? elements;
+|};
+
 # Represents OBJECT TYPE Oracle DB field.
 #
 # + value - Value of parameter passed into the SQL statement
```

Comment:
Do we use these object types anywhere

Reply:
Object types are only used to pass the parameters to the prepared statement. when retrieving data we are not using object types.